config/urls.py

```diff
@@ -78,6 +81,15 @@
     VersionDetail,
 )
 
+djdt_urls = []
+try:
+    if DEBUG_TOOLBAR:
+        from debug_toolbar.toolbar import debug_toolbar_urls
+
+        djdt_urls = debug_toolbar_urls()
+except ModuleNotFoundError:
+    logging.error("DEBUG_TOOLBAR enabled, debug_toolbar not installed. Run just build.")
```

Comment:
It took me way too long to understand the second sentence 😅

```suggestion
    logging.error("DEBUG_TOOLBAR enabled; debug_toolbar not installed. Run `just build`.")
```

Reply:
Yeah, fair point, it's not very clear. Have adjusted it.